R/getAge.R

```diff
@@ -1,131 +1,127 @@
-#' For example, `keep = c("B", "L")` will keep all `"BB"`, `1`, and `6` entries as well.
-#' * "B" == 1, "BB", or break and burn
-#' * "S" == 2, "SR" (California), or surface read
-#' * 3 == scales (Oregon data)
-#' * "T" == 4, "TS" (California), "X" (Washington), or thin section
-#' * "O" == 5 or optical scanner
-#' * "L" == 6 or age derived from length
-#' * 9 == unable to read or age
-#' * U == unknown
+#' How `FISH_AGE_YEARS_FINAL` is determined can be unclear because it doesn't
+#' always equal the value from the first age reader. Sometimes, it appears to
+#' default to break and burn. We use the best judgment of those who created the
+#' column that they know what they are doing; but, please feel free to report
+#' errors in PacFIN to GitHub issues or the package maintainer after contacting
+#' the state representative. Previously, `getAge()` averaged over multiple reads
+#' if there was no entry in `FINAL_FISH_AGE_IN_YEARS`. This functionality has
+#' since been removed from the package starting with version 0.2.6.
 #'
-#' The final age assigned to a fish when there are multiple age reads depends on the
-#' ageing lab. See issue #49 for a semi-detailed discussion. Sometimes, reconciling
-#' happens where the best age will not be equal to any age read from a specific reader
-#' but the readers get together to determine the best read.
-#' Washington uses the following order of operations to assign the best age given
-#' there are multiple reads. Note, that they will never report more than three reads
-#' because that is the limitation of their database.
+#' The input argument of `keep` can contain a mix of numeric and character
+#' values, but all values will be converted to character values. See
+#' [codify_age_method()] for how characters and integers are translated to the
+#' returned characters. Only at least one age read method needs to have been
+#' used to keep the entry in `FINAL_FISH_AGE_IN_YEARS`.
+#'
+#' The final age assigned to a fish when there are multiple age reads depends on
+#' the ageing lab. See [issue
+#' #49](http://www.github.com/pfmc-assessments/PacFIN.Utilities/issues/49) for a
+#' semi-detailed discussion. Sometimes, reconciling happens where the best age
+#' will not be equal to any age read from a specific reader but the readers get
+#' together to determine the best read. Washington uses the following order of
+#' operations to assign the best age given there are multiple reads. Note, that
+#' they will never report more than three reads because that is the limitation
+#' of their database.
 #' * age1 if only one read;
-#' * randomly chosen age between age-read 1 and age-read 2 if the difference between
-#' the two reads is less than three;
-#' * if the difference between two reads is more than three, then the best age is
-#' manually keyed in from that designated by a human;
+#' * randomly chosen age between age-read 1 and age-read 2 if the difference
+#'   between the two reads is less than three;
```

Comment:
According to http://www.github.com/pfmc-assessments/PacFIN.Utilities/issues/49 it is less than **or equal to** three

Reply:
Thanks @brianlangseth-NOAA, I also went back to the original email from Theresa to double check and you are right.